fluss-client/src/main/java/com/alibaba/fluss/client/metrics/ScannerMetricGroup.java

```diff
@@ -55,6 +59,7 @@ public class ScannerMetricGroup extends AbstractMetricGroup {
     private volatile double pollIdleRatio;
     private volatile long lastPollMs;
     private volatile long pollStartMs;
+    private final Map<Integer, BucketMetricGroup> buckets = new HashMap<>();
```

Comment:
We can't rely on `BucketMetricGroup` in client module, because it is in server module. See the build error:

```
Error:  COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
Error:  /home/fluss/actions-runner/_work/fluss/fluss/fluss-client/src/main/java/com/alibaba/fluss/client/metrics/ScannerMetricGroup.java:[34,46] package com.alibaba.fluss.server.metrics.group does not exist
Error:  /home/fluss/actions-runner/_work/fluss/fluss/fluss-client/src/main/java/com/alibaba/fluss/client/metrics/ScannerMetricGroup.java:[62,32] cannot find symbol
  symbol:   class BucketMetricGroup
  location: class com.alibaba.fluss.client.metrics.ScannerMetricGroup
Error:  /home/fluss/actions-runner/_work/fluss/fluss/fluss-client/src/main/java/com/alibaba/fluss/client/metrics/ScannerMetricGroup.java:[130,43] cannot find symbol
  symbol:   class BucketMetricGroup
  location: class com.alibaba.fluss.client.metrics.ScannerMetricGroup
```

https://github.com/alibaba/fluss/actions/runs/12250266479/job/34187811565?pr=157

Reply:
so can i use GenericMetricGroup instead of it ?